test/axes/baseAxisTests.ts

```diff
@@ -29,11 +29,11 @@ describe("BaseAxis", () => {
     var verticalAxis = new Plottable.Axis(scale, "right");
     verticalAxis.renderTo(svg);
 
-    var expectedWidth = verticalAxis.tickLength() + verticalAxis.margin(); // tick length and margin by default
+    var expectedWidth = verticalAxis.innerTickLength() + verticalAxis.margin(); // innerTickLength and margin by default
```

Comment:
the comment was fine as it was. The reader of this comment doesn't particularly care if it's the inner tick length (actually the reader would wonder why is the inner tick more special)